webapp/store/views.py

```diff
@@ -186,6 +187,8 @@ def details_overview(entity_name):
 
     readme = md_parser(readme)
     readme = decrease_headers(readme)
+    if package["type"] == "bundle":
+        readme = inject_topology_image(readme, package["name"])
```

Comment:
I think this is not necessary for now.
Let's keep the page simple. We can't get the topology now, let's just show the readme.